Demo/src/main/res/xml/settings.xml

```diff
@@ -179,6 +179,12 @@
             android:summary="@string/venmo_fallback_to_web_summary"
             android:defaultValue="false" />
 
+        <CheckBoxPreference
+            android:key="venmo_fallback_to_web"
+            android:title="@string/venmo_fallback_to_web"
+            android:summary="@string/venmo_fallback_to_web_summary"
+            android:defaultValue="false" />
+
```

Comment:
Do we need this anymore since it's the default behavior?

Reply:
Ah no - removed